docs/source/error_codes.rst

```diff
@@ -37,8 +37,8 @@ Silencing errors based on error codes
 You can use a special comment ``# type: ignore[code, ...]`` to only
 ignore errors with a specific error code (or codes) on a particular
 line.  This can be used even if you have not configured mypy to show
-error codes. Currently it's only possible to disable arbitrary error
-codes on individual lines using this comment.
+error codes. Also you can use :option:`--ignore-error-codes <mypy --ignore-error-codes>`
+for more coarse-grained (per file or per directory) silencing of errors.
```

Comment:
The actual command line flag only allows global silencing of errors, but this implies that it's per file or per directory. Either leave the parenthetical remark out, or somehow make it clear that per-file/directory options are only available through the config file or `# mypy: ...` comments. For consistency, I'd leave it out, since we probably don't want to discuss this separately for every reference to every option.